packages/paste-website/src/components/PropsTables.tsx

```diff
@@ -0,0 +1,171 @@
+const PropType: React.FC<React.PropsWithChildren<PropTypeProps>> = ({children}) => {
+  return (
+    <Box as="code" fontFamily="fontFamilyCode" color="colorTextDecorative30">
+      {children}
+    </Box>
+  );
+};
+PropType.displayName = 'PropType';
+
+const PropPair: React.FC<PropPairProps> = ({term, description}) => {
+  return (
+    <Box display="flex" marginBottom="space20" columnGap="space20" paddingX="space30">
+      <Box as="dt" fontSize="fontSize20" fontWeight="fontWeightMedium" minWidth="80px">
+        {term}
+      </Box>
+      <Box as="dd" fontSize="fontSize20" margin="space0">
+        {description}
+      </Box>
+    </Box>
+  );
+};
+PropPair.displayName = 'PropPair';
+
+const PropsList: React.FC<PropsListProps> = ({props}) => {
+  return (
+    <Box>
+      {Object.keys(props).map((propName) => {
+        const prop: ComponentAPIPropDetails = props[propName];
+        return (
+          <React.Fragment key={propName}>
+            <Box marginBottom="space60">
+              <Box
+                as="h4"
+                fontWeight="fontWeightMedium"
+                marginBottom="space30"
+                display="flex"
+                alignItems="center"
+                columnGap="space50"
+              >
+                <Box
+                  as="span"
+                  color="colorTextDecorative30"
+                  fontFamily="fontFamilyCode"
+                  fontSize="fontSize30"
+                  backgroundColor="colorBackgroundDecorative30Weakest"
+                  borderStyle="solid"
+                  borderWidth="borderWidth10"
+                  borderColor="colorBorderDecorative30Weaker"
+                  paddingY="space20"
+                  paddingX="space30"
+                  lineHeight="lineHeight10"
+                  borderRadius="borderRadiusPill"
+                >
+                  {propName}
+                </Box>{' '}
+                {prop.required ? (
+                  <Box as="span" display="flex" alignItems="center" columnGap="space30">
+                    <RequiredDot />
+                    <Text as="small" fontSize="fontSize20" fontFamily="fontFamilyText">
+                      Required
+                    </Text>
+                  </Box>
+                ) : (
+                  ''
+                )}
+              </Box>
+              {prop.description && (
+                <Box as="p" fontSize="fontSize20" margin="space0" paddingX="space30">
+                  <MarkdownConverter description={prop.description} />
+                </Box>
+              )}
+            </Box>
+            <Box as="dl">
+              <PropPair term="Type" description={<PropType>{prop.type}</PropType>} />
+              {prop.defaultValue && prop.defaultValue !== ' ' ? (
+                <PropPair term="Default" description={<PropType>{prop.defaultValue}</PropType>} />
+              ) : null}
+            </Box>
+            <Separator orientation="horizontal" verticalSpacing="space120" />
```

Comment:
suggestion: on the last separator for each component, drop the bottom margin to space-50 (1 rem) so that there's less spacing between the separator and Disclosure for inherited props.

Reply:
Technically I can't do that on separators, it's uniform top and bottom spacing.

Reply:
womp womp ok fine, ignore my other spacing comment too

Reply:
actually, don't ignore it quite yet. i still think there's too much spacing between "Props" and the first heading underneath it. lemme play around with it.

Reply:
Reinvented it with a border on Box

Reply:
lol ok unedited my other comment